src/cmd/troubleshoot/image_pullable.go

```diff
@@ -70,14 +67,27 @@ func checkOneAgentImagePullable(troubleshootCtx *troubleshootContext) error {
 	}
 
 	dynakubeOneAgentImage := getOneAgentImageEndpoint(troubleshootCtx)
-	if err != nil {
+
+	if err = checkComponentImagePullable(troubleshootCtx.httpClient, "OneAgent", pullSecret, dynakubeOneAgentImage); err != nil {
 		return err
 	}
 
-	if err = checkComponentImagePullable(troubleshootCtx.httpClient, "OneAgent", pullSecret, dynakubeOneAgentImage); err != nil {
+	return nil
+}
+
+func checkOneAgentCodeModulesImagePullable(troubleshootCtx *troubleshootContext) error {
+	logNewTestf("checking if OneAgent codeModules image is pullable ...")
+
+	pullSecret, err := getPullSecretToken(troubleshootCtx)
+	if err != nil {
 		return err
 	}
 
+	if dynakubeOneAgentCodeModulesImage := getOneAgentCodeModulesImageEndpoint(troubleshootCtx); dynakubeOneAgentCodeModulesImage != "" {
+		if err := checkCustomModuleImagePullable(troubleshootCtx.httpClient, "OneAgentCodeModules", pullSecret, dynakubeOneAgentCodeModulesImage); err != nil {
+			return err
+		}
+	}
```

Comment:
```suggestion
	dynakubeOneAgentCodeModulesImage := getOneAgentCodeModulesImageEndpoint(troubleshootCtx)
	err := checkCustomModuleImagePullable(troubleshootCtx.httpClient, "OneAgentCodeModules", pullSecret, dynakubeOneAgentCodeModulesImage)
	if err != nil {
		return err
	}
```

so that it would look the same as in line 72-75 in `checkOneAgentImagePullable`